website/www/site/content/en/documentation/sdks/python-type-safety.md

```diff
@@ -90,6 +91,15 @@ The following code declares an `int` input and a `str` output type hint on the `
 {{< code_sample "sdks/python/apache_beam/examples/snippets/snippets_test_py3.py" type_hints_map_annotations >}}
 {{< /highlight >}}
 
+The following code demonstrates how to use annotations on `PTransform` subclasses. 
+A valid annotation is a `PCollection`, `PBegin`, or `PDone` that wraps an internal (nested) type. 
```

Comment:
Good question. Responded on the list.

Reply:
The dev list [discussion](https://lists.apache.org/thread.html/r5cad76ea7b5b8404d159dd14c24964242b1d4b49787d6aa6c162d948%40%3Cdev.beam.apache.org%3E) seems to have a consensus:
- Make PBegin public (add to \_\_all\_\_ list)
- Deprecate PDone return type in favor of None. We still need to be backwards compatible, but update this PR to support the None return type for expand.

Additional documentation will be tracked in https://issues.apache.org/jira/browse/BEAM-10495

Reply:
Sounds good. 

Should we update documentation [here](https://beam.apache.org/documentation/programming-guide/) and [here](https://beam.apache.org/documentation/sdks/python-type-safety/) to note the existence of `PBegin`?

Reply:
I would like to split the programming guide update into a separate PR, since it will include examples from multiple languages and possibly some more discussion. This will be tracked in BEAM-10495, and you can take it on if you wish.

The python-type-safety update in this PR already mentions PBegin. Did you want to add more details?

Reply:
Sounds good

> The python-type-safety update in this PR already mentions PBegin. Did you want to add more details?

It also mentions PDone, do we want to remove that?

> I would like to split the programming guide update into a separate PR, since it will include examples from multiple languages and possibly some more discussion. This will be tracked in [BEAM-10495](https://issues.apache.org/jira/browse/BEAM-10495), and you can take it on if you wish.

I'll focus on benchmarking the runtime type checking system for now but if time permits afterwards, sure

Reply:
Please also accept `None` wherever `PDone` is accepted, so as to not break existing pipelines.